src/pat/modal/modal.js

```diff
@@ -31,7 +31,7 @@ define([
         _init_inject1: function () {
             var opts = {
                 target: "#pat-modal",
-                "class": "pat-modal" + (this.options["class"] ? " " + this.options["class"] : "")
+                "class": "pat-modal modal-active" + (this.options["class"] ? " " + this.options["class"] : "")
             };
```

Comment:
Please make `modal-active` the default value of the `class` parser argument, then it will be added automatically above and it doesn't have to be hardcoded.